packages/api/src/controllers/asset.ts

```diff
@@ -361,6 +361,12 @@ export async function withPlaybackUrls(
     console.error("Error getting asset object store", err);
     return asset;
   }
+  if (asset.sourcePlaybackReady) {
```

Comment:
An alternative could be also to just use the existence of `files` in the asset, specifically for a playback manifest. Those are only set on termination today and are now being set in the middle of processing. Not required change just wondering, WDYT?

Reply:
Yeah I was thinking the same really. Does this work: https://github.com/livepeer/studio/pull/1748/commits/a88980c404b69e9ace01d3ecfa8a7fad638e6d5d

Reply:
Actually no if it's undefined then it'll be hidden from the response object anyway

Reply:
Yeah like that exactly! Didn't understand your following comment did it have any problems?

Reply:
Yep worked ok, but yes i'm still using the bool in one other place so will do a follow up PR